<?xml version="1.0"?>

<launch>
  <!-- Navigation Master -->
  <node pkg="navigation_master" type="navigation_master_v1_6.py" name="navigation_master_v1_6" output="screen">
    <param name="path_directory" value="$(find atlas80evo)/traj/psa"/>
    <param name="mission_sound" value="$(find atlas80evo)/sounds/siren_beeping.ogg"/>
    <param name="default_sound" value="$(find atlas80evo)/sounds/ASV_Standby_01.ogg"/>
    <param name="battery_threshold" value="20"/> <!-- [%] -->
  </node>

  <!-- Doing Table Pickup Action -->
  <node pkg="navigation_master" type="action_pickup_table_v2.py" name="action_pickup_table_v2" output="screen"/>
  <node pkg="under_table" type="table_finder_v3.py" name="table_finder_v3"/>

  <!-- Doing Table Dropoff Action -->
  <node pkg="navigation_master" type="action_dropoff_table_v2.py" name="action_dropoff_table_v2" output="screen"/>
  <node pkg="turning_180" type="turning_180_v4.py" name="turning_180_v4"/>
  <node pkg="under_table" type="leave_table_v3_2.py" name="leave_table_v3_2"/>

  <!-- Charging Action -->
  <node pkg="navigation_master" type="action_charging_v2_2.py" name="action_charging_v2" output="screen">
    <param name="battery_full" value="100"/>
    <param name="full_charge_time" value="200"/>
    <param name="start_charge_time" value="1"/>
  </node>
  <node pkg="self_charging" type="charger_finder_v3.py" name="charger_finder_v3"/>

</launch>
